telluric/vectors.py

```diff
@@ -301,13 +301,21 @@ def to_geojson(self, filename):
         with open(filename, 'w') as fd:
             json.dump(self.to_record(WGS84_CRS), fd)
 
+    @classmethod
+    def empty(cls):
```

Comment:
where do we need it?

Reply:
It was explicitly requested in #114 by @slava-kerner.

Reply:
thanks :)